doc/helpers.cmake

```diff
@@ -38,6 +38,7 @@ function(doc_gen name in pdf mainpage)
   add_custom_target("${name}" ALL
     #DEPEND "${doxyfile}.stamp" Doxyfile.in ${in} ${in2}
     COMMAND "${DOXYGEN_EXECUTABLE}" "${doxyfile}"
+            > "${CMAKE_CURRENT_BINARY_DIR}/${name}.log" 2>&1
```

Comment:
There is an option `WARN_LOGFILE` see [doxygen doc](https://www.stack.nl/~dimitri/doxygen/manual/config.html#cfg_warn_logfile) maybe that is cleaner than doing it in CMake.